staff/RominaFior/blog/index.html

```diff
@@ -0,0 +1,123 @@
+                        reglas. Lamentablemente, según he podido observar, estas pocas reglas no siempre se tienen en cuenta,
+                        en especial cuando la dieta vegana se lleva a cabo por motivos éticos. Sin embargo, precisamente
+                        quien es vegano por motivos éticos debería alimentarse de forma saludable para poder constituir un
+                        ejemplo positivo del veganismo y motivar a otras personas a llevar este tipo de alimentación. Los
+                        ejemplos negativos suelen provocar reacciones contra el veganismo.
+                    </p>
+                </header>
+            </article>
+            <article id="entradas">
+
+                <header>
+                    <h3>Medio Ambiente</h3>
+                    <h4>11/04/2018 RominaFior Planeta</h4>
+                    <img src="images/planet.png" alt="planeta" width="250">
+                    <p> Como hemos visto, el usar animales para alimentarnos hace que también tengamos que producir los vegetales
+                        para mantenerlos. Como además se necesita más cantidad de vegetales para alimentarlos que la que
+                        consumiríamos nosotros directamente, el desperdicio de comida, degradación del terreno, contaminación,
+                        etc es mucho mayor. Hay que tener en cuenta, también, que los animales de por sí son mucho más contaminantes
+                        que los vegetales, por lo que parece claro que el ser vegano es muy beneficioso para el medio ambiente
+                        y, por lo tanto, para todos los animales.
+                    </p>
+                </header>
+            </article>
+            <article>
+                <header>
+
+                    <h3>Filosofía</h3>
+                    <h4>11/04/2018 RominaFior Naturaleza</h4>
+                    <img src="images/design-3289964_640.png" alt="naturaleza" width="250">
+                    <p>El veganismo es un estilo de vida basado en el respeto hacia los animales. Considerar que los animales
+                        son individuos con intereses que merecen ser respetados tiene implicaciones prácticas que evitan
+                        que miles de ellos sufran y/o mueran cada segundo en mataderos, centros de experimentación, zoos,
+                        acuarios, circos, granjas peleteras, etc.
+                    </p>
+                </header>
+            </article>
+        </section>
+        <section>
+            <a id="comentarios">
+                <h4>COMENTARIOS</h4>
+            </a>
+            <textarea name="comentarios" id="comentarios" cols="30" rows="10"></textarea>
+        </section>
+    </main>
+    <aside>
+        BUSCAR ESTE BLOG
+        <br>
+        <input type="text" value="" placeholder="">
+        <button>Buscar</button>
+        <br> SEGUIDORES
+        <br>
+        <input type="text" value="" placeholder="">
+        <button>Seguir</button>
```

Comment:
WRAP input and button inside a form tag. SEE https://www.w3schools.com/html/html_forms.asp

Reply:
done!